doc/nrf/ug_radio_coex.rst

```diff
@@ -0,0 +1,196 @@
+   * ``pri-dir-gpios`` - GPIO characteristic of the device that controls the ``PRIORITY`` signal of the PTA (priority + RX/TX direction).
+   * ``grant-gpios`` - GPIO characteristic of the device that controls the ``GRANT`` signal of the PTA (RF medium access granted).
+
+   Type ``phandle-array`` is used here, which is common in Zephyr's devicetree to describe GPIO signals.
+   The first element ``&gpio0`` refers to the GPIO port ("port 0" has been selected in the example shown).
+   The second element is the pin number on that port.
+
+#. On nRF53 devices, you must also apply the same devicetree node mentioned in step 2 to the network core.
+   To do so, apply the overlay to the correct network core child image by creating an overlay file named :file:`child_image/*childImageName*.overlay` in your application directory, for example :file:`child_image/multiprotocol_rpmsg.overlay`.
+
+   The ``*childImageName*`` string must be one of the following values:
+
+   *  ``multiprotocol_rpmsg`` for multiprotocol applications having support for both 802.15.4 and Bluetooth.
+   *  ``802154_rpmsg`` for applications having support for 802.15.4, but not for Bluetooth.
+   *  ``hci_rpmsg`` for application having support for Bluetooth, but not for 802.15.4.
+
+
+
+WiFi Coexistence implementations
+********************************
+
+The following CX implementations are available:
+
+* :ref:`Generic 3-pin interface`
+* :ref:`"Thread" implementaion`
+
+Each implementaion is a plugin consisting of one or more C source files interfacing with the MPSL API (which in turn communicates with the
+SR protocol drivers).
+
+When either CX variant is enabled, the driver requests access to the RF medium from the PTA and informs it about the operation it's about
+to perform. It also reacts properly to the information from the PTA that access to RF medium was granted/denied.
+
+Generic 3-pin interface
+=======================
+
+This implementation uses the following pins to communicate with the PTA:
+
+* REQUEST pin - output controlled by the protocol driver, it is asserted in order to request RF access from the PTA. It can be configured by
+  setting the `req-gpios` property of the `nrf_radio_coex` device tree node.
+* PRIORITY pin - output controlled by the protocol driver, it contains the information about the the type of operation
+  (RX/TX) to perform. It can be configured by setting the `pri-dir-gpios` property of the `nrf_radio_coex` device tree node.
+* GRANT pin - output controlled by the PTA, asserts, when PTA grants access to RF to the 802.15.4 and deasserts when it denies the access. It can
+  be configured by setting the `grant-gpios` property of the `nrf_radio_coex` device tree node.
+
+The support for this interface is provided in a single-file plugin located in the sdk-nrf repo, in the subsys/mpsl/cx directory.
+It is only available to use with the Zephyr RTOS.
```

Comment:
`It is only available to use with the Zephyr RTOS.` please remove this (and for "Thread" version also). Whole NCS is Zephyr-based, there is no cx plugin without NCS and Zephyr.

Reply:
> the user can create a plugin with/or without zephyr if he wishes,

We don't document in the file `doc/nrf/ug_radio_coex.rst` many of the aspects related to nRF Connect SDK and Zephyr. This includes for example what compiler should be used. What if an user decides to implement plugin in Python or C#? 
Should we write that this plugin supports C99 compiler only? 
What if an user decides to write a plugin which does not follow devicetree. Should we write that "It is only available to use with devicetree". ?

Our deliverable works in context of nRF Connect SDK which is Zephyr-based.

Please remove this sentence. It brings more noise than value.

Reply:
deleted